cmake/Install.cmake

```diff
@@ -363,6 +363,7 @@ if(LINK_WITH_PYTHON)
 endif()
 set(CPACK_RESOURCE_FILE_README "${PROJECT_SOURCE_DIR}/release/readme.html")
 
+install(FILES "${PROJECT_SOURCE_DIR}/bin/CurveFitTools/CurveFitTool.xlsm" DESTINATION "PreProcess/HVACCurveFitTool/")
```

Comment:
Install `CurveFitTool.xlsm` on all platforms, just like `IceStorageCurveFitTool.xlsm` is already installed on all platforms.